test/activities/quizzes/QuizEntity.js

```diff
@@ -344,4 +344,18 @@ describe('QuizEntity', () => {
 			expect(fetchMock.done());
 		});
 	});
+
+	describe('previewHref', () => {
+		const testPreviewHref = 'http://test.desire2learn.d2l/d2l/lms/quizzing/user/quiz_summary.d2l?ou=6606&qi=46&isprv=1&fromQB=1&bp=1';
+
+		it('can read previewHref from an editable entity', () => {
+			var quizEntity = new QuizEntity(editableEntity);
+			expect(quizEntity.previewHref()).to.equal(testPreviewHref);
+		});
+
+		it('can read previewHref from a non-editable entity', () => {
```

Comment:
The outcome for the `previewHref` isn't determined if an entity is editable or not but it probably is good to keep these 2 tests anyway?

Reply:
Yeah I mean the second one seems pretty redundant right now...I think it will be updated to something more meaningful once we add the permissions check